File: sensor1.cpp
#include <iostream>
#include <string>
using namespace std;

class Valvula {
public:
    string tag;
    double abertura;

    Valvula(string novaTag, double aberturaInicial) {
        tag = novaTag;
        abertura = aberturaInicial;
    }

    void exibirEstado() {
        cout << tag << " -> " << abertura << "%\n";
    }
};

int main() {
    Valvula v1("XV-02", 35.0);
    v1.exibirEstado();
    return 0;
}

File: sensor1.py
class Valvula:
    def __init__(self, tag, abertura):
        self.tag = tag
        self.abertura = abertura

    def exibir_estado(self):
        print(f"{self.tag} -> {self.abertura}%")


v1 = Valvula("XV-02", 35.0)
v1.exibir_estado()